### PR TITLE
Webhook snapshotrollback

### PR DESCRIPTION
**Pull request by itaygvlz**

### CODE REVIEW

File: charts/volumez-csi/values.yaml [hunk 1]
```diff
@@ -1,28 +1,45 @@
...
...fgvyBmDDmfVc3h1Hzd0Vwvy2Vds-JDQE8dYYzlVItRSIO7UAIrPVbz9U8xTk1KsuegvpovO5ZVGd4zbijV0Fg2Tfxcgbe_IpZ4gGhIcUIF59l2i25eJHGjMvfEVNx4oSPT_eKS3lNywaoesLNBVuK9Nhf1q1v3_cEhbo6qGJb9wcSST8-vbo83h8w811SXFEfPBLfqfdbxWZdnXbGo7Ji_6C3FYtpVhmKMLbLWlHS-wuEt-MgVenVpX-m9NRSRPMbxHVB5SYAn_wtQhqVAo8pXZTSAfHjcTw2KBFmoFc-D12_T-fxPbN_O1Es4ddqg2GZZDUJEbQG212RvlZtLqatKTr6WaqBP9zCYjK1VR-NWX2djGdS1XBqLr_Ew3YWh8yQAIgxcFxSXUxIy5U9593jwdHJPiEKPzgDyRf2refOnJaNv_0PmX1E3HnUreYJzpo-69zK2bhWxWWwnh1Dge6IRhrLrqK-cVuhWsLm_lGIendt_Zxw_ksxMFFVYjrmhExDqW5ijjEY8JrbAP0UE-dCgEqBoXcux6WYSqaIu9iMc_h8icsPwZ1EgaVMNMednWIsJF_FjxLESUfph36bfDBsMla1v6JdPoNxorShUykSjMR0gBOd1RSBuUUjj-Ba4WS_GbQnscE6hbBnyVCqvZ5PKS5QFNZpv7ziOzFQLxpXGhwdFfcgJtRWOVnx4XB_rU0bWlwj4czO5p2CGzGBG2WlWaEMWCk2RaZnQPQohSxFWXbcS4zKBP5p2yR3oi-G9MAGkD8rwucnaCD_Z1xo4j_J3G0JJZcHpEtMRwyFssiGktaOeePTi-SR2ZfsUUzeLKc6or45RthR8km1299_aRtAHzAHvjFPpc_opxH-4ewSd2bTnuRrUbXhcB78YJUb7TjfCcPZJWshf2Ewq8RBQxN8FjubJRAxvqUcbL-MYBhx05Bi92YKnvDoPJ72KM5nh787oevTa_Hp1Htd08NlLNE3PMWpUy1zyWQzN_JxaKLdE5AiqV4pksFxz3_C2_-vunlqHx1ANAylK9ITS0kuUgaQR2yP7h028hvyhCVUBm3vQlLGZS8x-PTaN2G771MyuFUW0I763fNfd-55GgFkXQrLMO44I8m3PeBC0XHjHkxdk5tiekdnM5MiK5OD5s8mSFZQ61rLlUVqS8qUnH2KvybwnjE.z5_xDGHxTs2VUTHxIsYppw"
```
Comment (VolumezTech) [inline charts/volumez-csi/values.yaml:28]: vlzAuthToken: "" # CSI Driver Token (Refresh Token)

File: charts/volumez-csi/values.yaml [hunk 2]
```diff
@@ -1,28 +1,45 @@
 # Default values for volumez-csi.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-envId: ""
-useKafkaCommunicator: false
+envId: "ayal-csi-Q9C0VS"
```
Comment (VolumezTech) [inline charts/volumez-csi/values.yaml:4]: empty

File: charts/volumez-csi/values.yaml [hunk 3]
```diff
@@ -1,28 +1,45 @@
...
+useKafkaCommunicator: true
```
Comment (VolumezTech) [inline charts/volumez-csi/values.yaml:5]: false

File: charts/volumez-csi/values.yaml [hunk 4]
```diff
@@ -1,28 +1,45 @@
...
-  websocketId: ""                 # <websocket-id>
+vlzAdmissionWebhook:
+  repository: public.ecr.aws/u0q8u2v6/vlz-admission-webhook
+  tag: latest
+
+vlzSnapshotRollbackController:
+  repository: public.ecr.aws/u0q8u2v6/vlz-snapshotrollback-controller
+  tag: latest
+
+vlzEndpoint:
```
Comment (VolumezTech) [inline charts/volumez-csi/values.yaml:19]: all back to default

File: charts/volumez-csi/values.yaml [hunk 5]
```diff
@@ -1,28 +1,45 @@
...
 
 vlzConnector:
-  repoUrl: https://signup.volumez.com/connector/ 
-  domain: connector.volumez.com   # empty if .vlzEndpoint.websocketId is set
+  repoUrl: https://volumez-qa-connector-repo.s3.amazonaws.com/connector/ayal
+  # empty if .vlzEndpoint.websocketId is set
```
Comment (VolumezTech) [inline charts/volumez-csi/values.yaml:32]: back to default

**Review by nkorobko — APPROVED** [review]

**Pull request merged**